### PR TITLE
Show cmds before running, link to console

### LINKED ISSUE #11: Feature Request: Allow Verbose Mode
Maybe when we first start running we display something like:
`Running in terse output mode - hit "v" to enter verbose mode`

In verbose mode we should show the `gcloud` commands we are running but not necessarily their output.

### LINKED ISSUE #38: Tell me how to remove it, or what happens if I don't
The script output tells me how to change the image and deploy a new version, but it doesn't tell me how to remove it.

It also doesn't tell me what happens if I don't - will it live forever? Will it expire?  Will I have to pay anything

### PR DESCRIPTION
**Pull request by ahmetb**
Fixes #38: We now explain app is billed only when handling requests, and we
link to Cloud Console to manage (i.e. delete) the deployed app.

Fixes #11: Verbose mode by printing the cmds before running them.

### CODE REVIEW

**Comment by ahmetb** [comment]
image

**Comment by ahmetb** [comment]
Some more text adjustment to make the end remarks look more brief:

image

**Comment by ahmetb** [comment]
Maybe I'll move "Your application is now live:" part to the last and keep the lines starting with `* ` in between "Successfully deployed" and that.

**Comment by ahmetb** [comment]
this also looks pretty good:

image

**Comment by jamesward** [comment]
Nice improvements!  Will make a few specific comments in the code.

File: cmd/cloudshell_open/main.go [hunk 1]
```diff
@@ -214,15 +214,20 @@ func run(c *cli.Context) error {
 	serviceName = tryFixServiceName(serviceName)
 
 	image := fmt.Sprintf("gcr.io/%s/%s", project, serviceName)
-	end = logProgress(fmt.Sprintf("Building container image %s...", highlight(image)),
-		fmt.Sprintf("Built container image %s.", highlight(image)),
+	fmt.Println(infoPrefix + " Will run command:")
```
Comment (jamesward) [inline cmd/cloudshell_open/main.go]:
```suggestion
	fmt.Println(infoPrefix + " FYI, running this command:")
```

Comment (ahmetb) [reply cmd/cloudshell_open/main.go]: I'll fix these altogether.

**Pull request merged**